lib/avalon/ffprobe.rb

```diff
@@ -26,35 +28,51 @@ def initialize(media_path)
         return
       end
       @json_output = JSON.parse(raw_output).deep_symbolize_keys
-      @video_stream = @json_output[:streams].select { |stream| stream[:codec_type] == 'video' }.first
+
+      @video_stream = json_output[:streams].select { |stream| stream[:codec_type] == 'video' }.first
+      @audio_stream = json_output[:streams].select { |stream| stream[:codec_type] == 'audio' }.first
     end
 
     def video?
-      # ffprobe treats plain text files as ANSI or ASCII art. This sets the codec type to video
-      # but leaves display aspect ratio `nil`. If display_aspect_ratio is nil, return false.
-      # ffprobe treats image files as a single frame video. This sets the codec type to video
-      # but the packet/frame count will equal 1. If packet count equals 1, return false.
-      return true if @video_stream && @video_stream[:display_aspect_ratio] && @video_stream[:nb_read_packets].to_i > 1
+      return true if video_stream
 
       false
```

Comment:
I think this could be simplified to:
```suggestion
      video_stream.present?
```

Reply:
Good call. Was being explicit when working on it to make sure I had the logic right but should have gone back through and simplified.